src/Symfony/Component/ErrorHandler/ErrorHandler.php

```diff
@@ -196,6 +194,12 @@ public function __construct(?BufferingLogger $bootstrappingLogger = null, bool $
         }, null, new class() extends \Exception {
         });
         $this->debug = $debug;
+
+        // BC layer for PHP < 8.4
+        if (\defined('E_STRICT')) {
+            $this->levels[\E_STRICT] = 'Runtime Notice';
+            $this->loggers[\E_STRICT] = [$bootstrappingLogger, LogLevel::WARNING];
```

Comment:
What about moving the whole block at the beginning of the method so that we do not set the logger manually but rely on the existing `setDefaultLogger()` method instead (and thus do not risk any regressions in case that method is modified in the future)?

Reply:
Yes! This will better match the current behavior